templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -199,7 +197,11 @@ if [[ "$TEST" = "docs" || "$TEST" = "publish" ]]; then
 {%- if plugin_name == 'pulpcore' %}
   pip install psycopg2-binary
 {%- else %}
-  cmd_prefix bash -c "cd pulpcore; pip install -r doc_requirements.txt"
+  if [[ "${RELEASE_WORKFLOW:-false}" == "true" ]]; then
+    cmd_prefix bash -c "pip install -r https://raw.githubusercontent.com/pulp/pulpcore/{{ pulpcore_branch }}/doc_requirements.txt"
```

Comment:
ugly hack

Reply:
I think we want to have pip install the plugin under test using the `--no-binary` option. As an example for pulpcore I think that would be `pip install --no-binary pulpcore pulpcore`

Reply:
when we try to install pulpocre this way, the CI fails, as it can't install cryptography
https://github.com/pulp/plugin_template/pull/680#discussion_r952783847